docs/run/quantum-serverless.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Run workloads with quantum serverless
+description: Run workloads using quantum serverless
+---
+
+# Run workloads with quantum serverless
+
+Premium users can run their workloads remotely on classical compute made available through IBM Quantum Platform.
+
+For more information, see the [quantum serverless docs](https://qiskit-extensions.github.io/quantum-serverless/).
+
+<Admonition type="note">
+    This is an experimental feature, subject to change.
+</Admonition>
+
+## Build a workflow
+
+Here is an example of computing the quasi-probability distribution of a quantum circuit using the Qiskit Runtime Estimator primitive. This Python script should be saved in your working directory. (Warning! All contents of the working directory will be shipped to the cluster for execution.)
```

Comment:
should be "computing the expectation value using the Qiskit Runtime Estimator primitive" since we've switched away from Sampler.